app.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+"""Task API Application
+
+This script sets up and runs a Flask-based API for managing tasks.
+"""
+from os import environ
+
+import connexion
+from dotenv import load_dotenv
+
+from login import login_blueprint
+from task import task_blueprint, db
+
+
+def create_app():
+    app = connexion.FlaskApp(__name__)
+    app.add_api('swagger.yaml')
+    application = app.app
+    load_dotenv()
+    application.config['SQLALCHEMY_DATABASE_URI'] = environ.get('DB_URL')
+    application.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = True
+    db.init_app(application)
+
+    with application.app_context():
+        db.create_all()
+
+    application.register_blueprint(task_blueprint)
```

Comment:
@YanaRudyi it is more common to divide single run factory function with several separate functions that are called inside of `create_app` factory. Why? Because this way you can make your code more modular and use Single Responsibility Principle from [SOLID](https://www.digitalocean.com/community/conceptual-articles/s-o-l-i-d-the-first-five-principles-of-object-oriented-design) approach. It is always better to decouple code to make it modular and reusable in several other places and/or change its run depending on the application configuration